Sources/MoveSingleAPISwift/Take/Take.swift

```diff
@@ -97,10 +103,6 @@ public actor Take: Identifiable, Equatable {
         try await self.moveFile.upload()
         try await self.createTake()
     }
-    
-    public func updateMetadata(_ metadata: [String: AnyHashable]?) async {
-        self.metadata = metadata
```

Comment:
@flxfischer by removing those lines we introduce breaking changes to the SDC app btw

Reply:
That is all right, because the old app is using the last version where it still works.